internal/cli/alerts/settings_fields_type.go

```diff
@@ -32,13 +32,16 @@ func (opts *ConfigFieldsTypeOpts) init() error {
 	return err
 }
 
+var matcherFieldsTemplate = `{{range .}} {{.}} {{end}}
+`
```

Comment:
Or
```suggestion
var matcherFieldsTemplate = `{{range .}}
{{.}}
{{end}}`
```

Reply:
This is the output:
```
TYPE_NAME

HOSTNAME

PORT

HOSTNAME_AND_PORT

REPLICA_SET_NAME

SHARD_NAME

CLUSTER_NAME
```

Let me know what you like most

Reply:
I like this one better 👍

Reply:
Let me know what you like between:

1)
```
TYPE_NAME

HOSTNAME

PORT

HOSTNAME_AND_PORT

REPLICA_SET_NAME

SHARD_NAME

CLUSTER_NAME
```

2)

```
TYPE_NAME
HOSTNAME
PORT
HOSTNAME_AND_PORT
REPLICA_SET_NAME
SHARD_NAME
CLUSTER_NAME
```

3)

```
 TYPE_NAME  HOSTNAME  PORT  HOSTNAME_AND_PORT  REPLICA_SET_NAME  SHARD_NAME  CLUSTER_NAME
```

Reply:
2.

Reply:
I like 2. 😸